my-app/src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```

Comment:
- [ ] You have deleted the package.json file. That file should not be deleted because that is where your dependencies and dev-dependencies live and that is how you start your project, so please include the package.json. Also, be reminded that the best way to start [building a new single-page application](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app) in React is with create-react-app so please use that one

Reply:
@GzimAsani Thank you for the suggestion , I think I used  create-react-app so please use that one isn't ?